Framework/PythonInterface/plugins/algorithms/LoadDNSLegacy.py

```diff
@@ -64,6 +64,9 @@ def PyInit(self):
 
         self.declareProperty(name="ElasticChannel",defaultValue=0,validator=IntBoundedValidator(lower=0),
                              doc="Time channel number where elastic peak is observed. Only for TOF data.")
+
+        self.declareProperty(name="Wavelength",defaultValue=0.0,validator=FloatBoundedValidator(lower=0.0),
```

Comment:
It is possible to use `EMPTY_DBL` as an indication of 'no value given'. I don't know which one would be nicer for the users, though.

Reply:
I think, zero should be easier to understand for users. Anyway, it is nonphysical value for the wavelength.